metashare/consumers.py

```diff
@@ -13,7 +13,7 @@
 Request = namedtuple("Request", "user")
 
 
-KNOWN_MODELS = {"user"}
+KNOWN_MODELS = {"user", "scratchorg"}
```

Comment:
It'd be nice if we could make this `scratch_org`, but I found a workaround.

Reply:
In addition to `model` and `id`, I'm now also passing in `action: 'SUBSCRIBE' | 'UNSUBSCRIBE'`, which currently causes the `is_valid` check to fail.